jsonschema.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+from collections import namedtuple
+
+Location = namedtuple(
```

Comment:
I think we should name this "Clinic" instead of location, to separate it from the concept of the geo location

Reply:
Gotcha. What about `facility` then as the generic term? Or the even more generic `place`?

Reply:
If these are called `Locations` elsewhere within VaccinateCA then I'll accept the ambiguity :)